### PR TITLE
Reworked tag implementation

### PR DESCRIPTION
**Pull request by OGNylux**

### CODE REVIEW

File: src/lib/helpers/mapbox.ts [hunk 1]
```diff
@@ -30,12 +30,13 @@ function filterFeatures(reponses: any, categories: string[]) {
...
 			const f = reponses[i].features[j];
-			results.push({name: f.properties.name, categories: f.properties.poi_category, address: f.properties.address, coordinate: {lng: f.geometry.coordinates[0], lat: f.geometry.coordinates[1]}, id: f.properties.mapbox_id, metadata: f.properties.metadata, maki: f.properties.maki, ismatch: false})
+			const inList = results.filter(e => e.id === f.properties.mapbox_id).length > 0;
+			if(results.length < pp && !inList) results.push({name: f.properties.name, categories: f.properties.poi_category, address: f.properties.address, coordinate: {lng: f.geometry.coordinates[0], lat: f.geometry.coordinates[1]}, id: f.properties.mapbox_id, metadata: f.properties.metadata, maki: f.properties.maki, isMatch: false})
 			full--;
 			inner--;
 		}
 	}
-	
+	console.log(results)
```
Comment (paulpaulsen55) [inline src/lib/helpers/mapbox.ts]: please remove unused console logs

File: src/app.css [hunk 2]
```diff
@@ -20,6 +20,9 @@
 		-ms-overflow-style: none;
 		scrollbar-width: none;
 	}
+	.active {
+		background-color: pink;
+	}
```
Comment (paulpaulsen55) [inline src/app.css]: what is this used for?

Comment (OGNylux) [reply src/app.css]: digga halt die schnauze wollte was testen

**Review by paulpaulsen55 — APPROVED** [review]
passt

**Pull request merged**